browser/profiles/brave_renderer_updater.cc

```diff
@@ -92,6 +100,21 @@ void BraveRendererUpdater::UpdateAllRenderers() {
     UpdateRenderer(&renderer_configuration);
 }
 
+const std::vector<std::string> BraveRendererUpdater::GetSkusAllowedOrigins() {
+  std::vector<std::string> result;
+  for (auto* const origin : kSkusAllowedOrigins) {
+    result.push_back(origin);
+  }
+  if (!skus_allowed_origin_for_testing_.empty())
```

Comment:
Should this check be done before appending to `result`? (just to save the extra loop)

Reply:
ah, nevermind- I mixed up something in my head. LGTM